test/dlc_tests/container_tests/bin/benchmark/run_pytorch_inductor_training_benchmark_gpu_huggingface

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+PYTHON_VERSION=$(python -c 'import sys; print(sys.version_info[0])' | tr -d "'")
+if [ "$PYTHON_VERSION" -eq 2 ]
+then
+  exit 0
+fi
+HOME_DIR=/test/benchmark
+BIN_DIR=${HOME_DIR}/bin
+LOG_DIR=${HOME_DIR}/logs
+
+mkdir -p ${HOME_DIR}
+mkdir -p ${BIN_DIR}
+mkdir -p ${LOG_DIR}
+
+python -c "import sys;
+assert sys.version_info < (3, 7);"
+PYTHON_VERSION_CHECK=`echo $?`
+
+set -e
+rm -rf $BIN_DIR/pytorch
+# tmp fix for numpy deprecations, PR raised: https://github.com/HewlettPackard/dlcookbook-dlbs/pull/19
+git clone https://github.com/pytorch/pytorch.git $BIN_DIR/pytorch
+
+pip install -U numpy
+pip install gitpython
+pip install deepspeed==0.8.2
+pip install tabulate==0.9.0
+
+
+TRAINING_LOG=${LOG_DIR}/pytorch_inductor_huggingface_benchmark.log
+
+rm -rf $BIN_DIR/pytorch/huggingface_logs
+mkdir -p $BIN_DIR/pytorch/huggingface_logs
+python $BIN_DIR/pytorch/benchmarks/dynamo/huggingface.py --performance --amp -dcuda --output=huggingface_logs/inductor_huggingface_amp_training_cuda_performance.csv --training --inductor --output-directory=$BIN_DIR/pytorch/ --no-skip --dashboard -x GPTJForCausalLM -x Reformer -x GPTJForQuestionAnswering -x GPTNeoForSequenceClassification -x BlenderbotForConditionalGeneration -x GPTNeoForCausalLM --cold-start-latency > $TRAINING_LOG 2>&1
```

Comment:
Why are we using this instead of `dynamo/runner.py --suites=huggingface` ? This runner presents a common interface to all 3 model suites and we do not have to update the command when the model coverage changes in upstream.

Reply:
These are the commands generated by runner.py, with the directory names modified to mitigate FileNotFound error. I find it technical to avoid it without leaving `container_tests/bin/benchmark`. So I used these codes. The model coverage should be reflected inside `huggingface.py`, so I guess we do not need to change the command later.